subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/versions/KotlinGradlePluginVersions.groovy

```diff
@@ -29,7 +33,25 @@ class KotlinGradlePluginVersions {
         '1.6.0', '1.6.10',
     ]
 
+    private static final VersionNumber KOTLIN_GRADLE_PLUGIN_1_5_0_VERSION_NUMBER = VersionNumber.parse('1.5.0')
+
     List<String> getLatests() {
         return LATEST_VERSIONS
     }
+
+    static void assumeCurrentJavaVersionIsSupportedBy(String kotlinPluginVersion) {
```

Comment:
It looks like we have a workaround for this in https://github.com/gradle/gradle/blob/c7fe43fbe09aa767d4de2d6dc249f851a541af22/subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinPluginSmokeTest.groovy#L232-L237

Reply:
I will use that workaround